### PR TITLE
added release notes file for 1.0.1 release

### PR DESCRIPTION
**Pull request by abhinavGupta16**
Signed-off-by: “Abhinav <“guptabhi123@gmail.com”>

### Description
Added the release notes file for 1.0.1 release
 
### Issues Resolved
N/A

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-release-notes-1.0.1.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.0.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.0.0.md).
+
+## BREAKING CHANGES
+
+## FEATURES
+
+## ENHANCEMENTS
+
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.0.1.md:18]: These can be removed

File: release-notes/opensearch-release-notes-1.0.1.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+* Fix index permissions for negative lookahead and negated regex index patterns ([#1300](https://github.com/opensearch-project/security/pull/1300))
+
+
+### Opensearch Security Dashboards Plugin
+* add resolve pemission to UI ([#803](https://github.com/opensearch-project/security-dashboards-plugin/pull/803))
+* Add refresh page to account-app.tsx to solve the tenant display issue ([#811](https://github.com/opensearch-project/security-dashboards-plugin/pull/811))
+
+
+## INFRASTRUCTURE
+
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.0.1.md:63]: remove this

File: release-notes/opensearch-release-notes-1.0.1.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+* Bump version to 1.0.1.0 ([#1418](https://github.com/opensearch-project/security/pull/1418))
+
+
+### Opensearch Security Dashboards Plugin
+* Add Integtest.sh for OpenSearch integtest setups ([#783](https://github.com/opensearch-project/security-dashboards-plugin/pull/783))
+* Bump plugin version to 1.0.1.0 ([#813](https://github.com/opensearch-project/security-dashboards-plugin/pull/813))
+
+
+## REFACTORING
+
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.0.1.md:105]: Remove this

File: release-notes/opensearch-release-notes-1.0.1.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+# OpenSearch and Dashboards 1.0.1 Release Notes
+
+## Release Highlights
+
+## Release Details
+
+OpenSearch and Dashboards 1.0.1 includes the following OpenSearch Migration, features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.0.1.md:7]: Change to
```
OpenSearch and Dashboards 1.0.1 includes the following OpenSearch bug fixes, documentation and maintenance updates.
```

File: release-notes/opensearch-release-notes-1.0.1.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+### Opensearch Security
+* Return HTTP 409 if get parallel put request ([#1158](https://github.com/opensearch-project/security/pull/1158))
+* Add validation for null array DataType ([#1157](https://github.com/opensearch-project/security/pull/1157))
+* Add support for ResolveIndexAction handling ([#1312](https://github.com/opensearch-project/security/pull/1312))
+* Fix LDAP authentication when using StartTLS ([#1415](https://github.com/opensearch-project/security/pull/1415))
+* Fix index permissions for negative lookahead and negated regex index patterns ([#1300](https://github.com/opensearch-project/security/pull/1300))
+
+
+### Opensearch Security Dashboards Plugin
+* add resolve pemission to UI ([#803](https://github.com/opensearch-project/security-dashboards-plugin/pull/803))
```
Comment (dblock) [inline release-notes/opensearch-release-notes-1.0.1.md:58]: Capitalize Add?

File: release-notes/opensearch-release-notes-1.0.1.md (hunk 1 above)
Comment (abhinavGupta16) [reply release-notes/opensearch-release-notes-1.0.1.md]: Removed

File: release-notes/opensearch-release-notes-1.0.1.md (hunk 2 above)
Comment (abhinavGupta16) [reply release-notes/opensearch-release-notes-1.0.1.md]: Removed

File: release-notes/opensearch-release-notes-1.0.1.md (hunk 3 above)
Comment (abhinavGupta16) [reply release-notes/opensearch-release-notes-1.0.1.md]: Removed

File: release-notes/opensearch-release-notes-1.0.1.md (hunk 4 above)
Comment (abhinavGupta16) [reply release-notes/opensearch-release-notes-1.0.1.md]: Changed

File: release-notes/opensearch-release-notes-1.0.1.md (hunk 5 above)
Comment (abhinavGupta16) [reply release-notes/opensearch-release-notes-1.0.1.md]: Changed.

File: release-notes/opensearch-release-notes-1.0.1.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+# OpenSearch and Dashboards 1.0.1 Release Notes
+
+## Release Highlights
+
+## Release Details
+
+OpenSearch and Dashboards 1.0.1 includes the following OpenSearch bug fixes, documentation and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.0.0.md).
```
Comment (dblock) [inline release-notes/opensearch-release-notes-1.0.1.md:9]: These should be `1.0.1.md`, no?

**Review by dblock — CHANGES_REQUESTED** [review]

File: release-notes/opensearch-release-notes-1.0.1.md (hunk 6 above)
Comment (bbarani) [reply release-notes/opensearch-release-notes-1.0.1.md:9]: We are not changing the -min version so we are not generating the new release notes for OpenSearch

File: release-notes/opensearch-release-notes-1.0.1.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
+# OpenSearch and Dashboards 1.0.1 Release Notes
+
+## Release Highlights
+
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.0.1.md:4]: Remove this per @elfisher thanks @abhinavGupta16

Comment (abhinavGupta16) [reply release-notes/opensearch-release-notes-1.0.1.md]: removed

Comment (peterzhuamazon) [reply release-notes/opensearch-release-notes-1.0.1.md]: Fixed.

**Review by gaiksaya — APPROVED** [review]
LGTM!

**Review by bbarani — DISMISSED** [review]

**Review by bbarani — DISMISSED** [review]

**Review by gaiksaya — APPROVED** [review]

**Review by bbarani — APPROVED** [review]

**Review by gaiksaya — APPROVED** [review]

**Pull request merged**